make

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#Operate in current direcory
+cd $(dirname $0)
+
+# Variables
+LANG="python3"
+MINIFY=0
+DEBUG=0
+
+#Ensure npm installed
+NPM="$(which npm)"
+if [ "$NPM" == "" ]; then
+	echo "npm not installed"
```

Comment:
They are spaces in my editor, since tabs are standard for bash, Github is displaying as tabs.